addon/README.md

```diff
@@ -113,14 +114,9 @@ window.addEventListener("from-addon-to-web", function (event) {
 
 Submit updates:
 ``` javascript
-sendToAddon({type: 'update-approve', detail: approvedUpdates});
+sendToAddon({type: 'install-experiment', detail: {xpi_url: 'https://people.mozilla.com/~jhirsh/universal-search-addon.xpi'}});
```

Comment:
just FYI, you want `https://people.mozilla.org/~jhirsch/universal-search-addon/addon.xpi`